packages/wallet-adapter-ant-design/src/WalletSelector.tsx

```diff
@@ -40,107 +73,133 @@ export function WalletSelector({
     }
   };
 
-  const onWalletSelected = (wallet: WalletName) => {
-    connect(wallet);
+  const closeModal = () => {
     setWalletSelectorModalOpen(false);
     if (setModalOpen) {
       setModalOpen(false);
     }
   };
-  const onCancel = () => {
-    setWalletSelectorModalOpen(false);
-    if (setModalOpen) {
-      setModalOpen(false);
-    }
-  };
-  const buttonText = account?.ansName
-    ? account?.ansName
-    : truncateAddress(account?.address);
+
+  const buttonText =
+    account?.ansName || truncateAddress(account?.address) || "Unknown";
```

Comment:
I've never seen it happen. I think this is mostly a lacking in our types. We only show this text if a wallet is connected and if `connected` is true then `account` is defined, but TypeScript doesn't understand that connection. I mostly just put this here to cover our bases since previously button text would otherwise be undefined.